build/stf-run-ci/tasks/setup_base.yml

```diff
@@ -123,7 +109,7 @@
         channel: beta
         installPlanApproval: Automatic
         name: prometheus
-        source: operatorhubio-operators
+        source: community-operators
```

Comment:
I'm going to review our STF 1.5 documentation because if we can avoid a whole set of steps using OperatorHub.io, I think that's better overall for the product. Also when installing from the UI you get a fancy little notice that these Operators aren't supported. (We don't use that method for installation though in the documentation so our users will never see it.)

Reply:
FWIW that's what I'm doing here: https://github.com/infrawatch/documentation/pull/406/files#diff-82772513f0a144752a726c46c7ab9d1431434ea047fa8dafa8d89bdf7b13baa7R199-R200

I copied it to both the Prometheus and the Elasticsearch subscription instructions.